src/gl/directx11/vcRenderShaders.cpp

```diff
@@ -1405,3 +1405,326 @@ const char *const g_udGPURenderGeomGeometryShader = R"shader(
     //OutputStream.RestartStrip();
   }
 )shader";
+
+const char *const g_FXAAVertexShader = R"shader(
```

Comment:
Can you put in a comment above the literal with the license information, please?